src/main/kotlin/racingcar/Race.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar
+
+class Race {
+    fun start(carCount: Int, roundCount: Int): List<Round> {
+        val carList = getCarList(carCount)
+        val roundList = getRoundList(roundCount)
+        for (currentRound in roundList) {
+            carList.map {
+                it.moveRandom()
+            }
+
+            currentRound.saveResult(carList)
+        }
+
+        return roundList
+    }
+
+    private fun getCarList(carCount: Int): List<Car> {
+        return List(carCount) { Car() }
```

Comment:
```suggestion
        return List(carCount) { Car() }.toList()
```
List 내부 코드를 보시면 MutableList로 되어있어요.
Immutable 타입인 List로 변환해서 리턴한다 할지라도, 외부에서 mutable로 형변환 후 수정을 하게 되면 원본에 영향을 미치게 돼요.
방어적 복사를 통해 원본에 영향을 미치지 않도록 변경해보면 어떨까요? (이하 관련 내용 동일)